src/Microsoft.Identity.Client/Exceptions/CoreErrorCodes.cs

```diff
@@ -75,6 +75,8 @@ internal class CoreErrorCodes
 
         public const string GetUserNameFailed = "get_user_name_failed";
         public const string NonParsableOAuthError = "non_parsable_oauth_error";
+        public const string HttpStatusNotFound = "not_found";
+        public const string HttpStatusCodeNotOk = "http_status_not_200";
```

Comment:
These have to get added to MsalErrorCodes

Reply:
will investigate.